packages/holocron/src/holocronModule.jsx

```diff
@@ -94,7 +94,7 @@ export default function holocronModule({
   shouldModuleReload,
   loadModuleData,
   mergeProps,
-  options = {},
+  options = { },
```

Comment:
```suggestion
  options = {},
```
I know this is dumb but it bothers me 😄